src/api/auth/LighthouseAuthApi.ts

```diff
@@ -19,12 +21,34 @@ interface ApiUser {
     permanent: boolean;
     updated_at: string;
   };
-  updated_at: string;
+}
+
+interface ApiToken {
+  api_token: string;
+  expires_at: string;
+  roles: [string];
```

Comment:
```suggestion
  roles: string[];
```

Unless we only intend to have a single role?

Reply:
Oopsie :smile: